### ISSUE TITLE
Zombie code clean up

### ISSUE THREAD

**Issue by pietercolpaert** [iRail/brail2gtfs-rt#23]
There is quite some code that isn't used. Can we remove this?

**Comment by brechtvdv** [comment]
Yep, the way it is build doesn't make a lot of sense. I would suggest to do a hackathon
